api/src/index.ts

```diff
@@ -65,26 +45,61 @@ async function run() {
     config.minterAddress
   );
 
-  const blockCursorService = new BlockCursorService();
+  // Make sure we're pointing to the correct Flow Access API.
+  fcl.config().put("accessNode.api", config.accessApi);
 
-  const saleOfferWorker = new SaleOfferHandler(
-    marketService,
-    blockCursorService,
-    flowService
-  );
+  const startWorker = () => {
+    const blockCursorService = new BlockCursorService();
 
-  const app = initApp(
-    knexInstance,
-    kibblesService,
-    kittyItemsService,
-    marketService
-  );
+    const saleOfferWorker = new SaleOfferHandler(
+      marketService,
+      blockCursorService,
+      flowService
+    );
 
-  app.listen(config.port, () => {
-    console.log(`Listening on port ${config.port}!`);
-  });
+    saleOfferWorker.run();
+  };
+
+  const startAPIServer = () => {
+    const kibblesService = new KibblesService(
+      flowService,
+      config.fungibleTokenAddress,
+      config.minterAddress
+    );
 
-  saleOfferWorker.run();
+    const kittyItemsService = new KittyItemsService(
+      flowService,
+      config.nonFungibleTokenAddress,
+      config.minterAddress
+    );
+
+    const app = initApp(kibblesService, kittyItemsService, marketService);
+
+    app.listen(config.port, () => {
+      console.log(`Listening on port ${config.port}!`);
+    });
+  };
+
+  if (argv.dev) {
+    // If we're in dev, run everything in one process.
+    startWorker();
+    startAPIServer();
+    return;
+  } else {
```

Comment:
This `else` block is redundant because the first case returns -- I would just remove to avoid a level indentation + nesting.

Reply:
yeah this is terrible.